app/kubernetes/views/applications/stats/stats.html

```diff
@@ -62,13 +64,13 @@
                 </select>
               </div>
               <span>
-                <pr-icon id="refreshRateChange" icon="'check'" feather="true" mode="'success'" size="'sm'"></pr-icon>
+                <pr-icon id="refreshRateChange" icon="'check'" mode="'success'" size="'sm'"></pr-icon>
```

Comment:
I'm pretty sure this should have `style="display: none"` like in EE https://github.com/portainer/portainer-ee/pull/2472/files#diff-ca2701e089fd3017c1463150929960dedf4c650b70e5944b61a7ad8602193691R67

Reply:
updated